tests/integ/test_session.py

```diff
@@ -586,7 +585,10 @@ def test_create_session_from_default_config_file(monkeypatch, db_parameters):
         with monkeypatch.context() as m:
             m.setenv("SNOWFLAKE_CONNECTIONS", tomlkit.dumps(doc))
             m.setenv("SNOWFLAKE_DEFAULT_CONNECTION_NAME", "default")
-            with Session.builder.create() as new_session:
+            if not IS_IN_STORED_PROC:
```

Comment:
Hmm now I'm not sure if it works (haven't tested this yet)

Reply:
https://ci-dev-142.int.snowflakecomputing.com/job/PythonStoredProcBuildPrecommitTest/895/
ran a SP test for you

Reply:
some tests failed. Looks like a rebase with main is needed before we do another run